tools/typst-preview-frontend/src/ws.ts

```diff
@@ -357,7 +357,8 @@ export async function wsMain({ url, previewMode, isContentPreview }: WsArgs) {
                 svgDoc.setPartialRendering(true);
                 return;
             } else if (message[0] === "invert-colors") {
-                const strategy = dec.decode((message[1] as any).buffer);
+                const rawStrategy = dec.decode((message[1] as any).buffer).trim();
+                const strategy = INVERT_COLORS_STRATEGY.find(t => t === rawStrategy) || (JSON.parse(rawStrategy) as StrategyMap);
```

Comment:
I fixed a bug about type (deserialization). Previously, the input is just raw string without deserialization.